.github/workflows/build.yml

```diff
@@ -19,38 +19,21 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.3"]
-                symfony: ["^5.4.21", "^6.4"]
-                sylius: ["~1.13.0", "~1.14.0"]
-                mysql: ["8.4"]
+                php: ["8.2", "8.3"]
                 node: ["20.x"]
+                mysql: ["8.4"]
```

Comment:
We should add builds to test postgres, but of course, in a separate PR, and starting from 1.14